### ISSUE TITLE
tensorflow lite only one output

### ISSUE THREAD

**Issue by iChiaGuo** [tensorflow/tensorflow#21116]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:yes
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:Linux Ubuntu 16.04
- **Mobile device (e.g iphone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:iphone 6s
- **TensorFlow installed from (source or binary)**:no
- **TensorFlow version (use command below)**:1.9
- **Python version**:3.5
- **Bazel version (if compiling from source)**:0.12
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

### Describe the problem
I use my model in tensorflowlite ,it have two outputs,but in my iphone6s only get one output!

(py35) ljg@ljg-linux:~/facenet/src/align/pb$ /home/ljg/tensorflow/bazel-bin/tensorflow/tools/graph_transforms/summarize_graph --in_graph=Pnet.pb
Found 1 possible inputs: (name=pnet/input, type=float(1), shape=[?,?,?,3]) 
No variables spotted.
Found 2 possible outputs: (name=pnet/conv4-1/BiasAdd, op=BiasAdd) (name=pnet/conv4-2/BiasAdd, op=BiasAdd) 
Found 6632 (6.63k) const parameters, 0 (0) variable parameters, and 0 control_edges
Op types used: 13 Const, 13 Identity, 6 Neg, 6 Relu, 5 BiasAdd, 5 Conv2D, 3 Add, 3 Mul, 1 MaxPool, 1 Placeholder
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=Pnet.pb --show_flops --input_layer=pnet/input --input_layer_type=float --input_layer_shape=-1,-1,-1,3 --output_layer=pnet/conv4-1/BiasAdd,pnet/conv4-2/BiasAdd

/home/ljg/tensorflow-1.9.0/bazel-bin/tensorflow/contrib/lite/toco/toco --input_file=Pnet.pb --input_format=TENSORFLOW_GRAPHDEF  --output_format=TFLITE --output_file=Pnet.tflite  --inference_type=FLOAT --inference_input_type=FLOAT --input_shape=1,12,12,3 --input_array=pnet/input --output_array=pnet/conv4-1/BiasAdd pnet/conv4-2/BiasAdd

![problem](https://user-images.githubusercontent.com/13581022/43180218-6e6282b0-9009-11e8-8bd4-f1d6046ba7e5.png)

### Source code / logs
    `class PNet(Network):
        def setup(self):
             (self.feed('data') 
             .conv(3, 3, 10, 1, 1, padding='VALID', relu=False, name='conv1')
             .prelu(name='PReLU1')
             .max_pool(2, 2, 2, 2, name='pool1')
             .conv(3, 3, 16, 1, 1, padding='VALID', relu=False, name='conv2')
             .prelu(name='PReLU2')
             .conv(3, 3, 32, 1, 1, padding='VALID', relu=False, name='conv3')
             .prelu(name='PReLU3')
             .conv(1, 1, 2, 1, 1, relu=False, name='conv4-1'))
             .softmax(3,name='prob1'))

             (self.feed('PReLU3') 
             .conv(1, 1, 4, 1, 1, relu=False, name='conv4-2'))
        `   
      `model_path='./'
      export_dir='./pb/'
      g1 = tf.Graph()
      sess1 = tf.Session(graph=g1)
      with sess1.as_default(): 
          with g1.as_default():
              with tf.variable_scope('pnet'):
                  data = tf.placeholder(tf.float32, (None,None,None,3), 'input')
                  pnet = PNet({'data':data})
                 pnet.load(os.path.join(model_path, 'det1.npy'), sess1)
            
    frozen_graphdef = tf.graph_util.convert_variables_to_constants(
          sess1, sess1.graph_def, ['pnet/conv4-2/BiasAdd', 'pnet/conv4-1/BiasAdd'])
    tf.train.write_graph(frozen_graphdef,export_dir, 'Pnet.pb', as_text=False)  
    print ('finis')
`

**Comment by freedomtan** [comment]
@iChiaGuo you cannot use `--output_array` to set multiple outputs, please use `--output_arrays` instead

**Comment by iChiaGuo** [comment]
thanks ,I solve my problem.
